testdata/p4_16_samples_outputs/psa-subtract-inst1.p4.spec

```diff
@@ -0,0 +1,94 @@
+
+struct ethernet_t {
+	bit<48> dst_addr
+	bit<48> src_addr
+	bit<16> ether_type
+}
+
+struct udp_t {
+	bit<16> sport
+	bit<16> dport
+	bit<16> length
+	bit<16> csum
+}
+
+struct psa_ingress_output_metadata_t {
+	bit<8> class_of_service
+	bit<8> clone
+	bit<16> clone_session_id
+	bit<8> drop
+	bit<8> resubmit
+	bit<32> multicast_group
+	bit<32> egress_port
+}
+
+struct psa_egress_output_metadata_t {
+	bit<8> clone
+	bit<16> clone_session_id
+	bit<8> drop
+}
+
+struct psa_egress_deparser_input_metadata_t {
+	bit<32> egress_port
+}
+
+header ethernet instanceof ethernet_t
+header udp_0 instanceof udp_t
+header udp_1 instanceof udp_t
+
+
+struct user_meta_data_t {
+	bit<32> psa_ingress_input_metadata_ingress_port
+	bit<8> psa_ingress_output_metadata_drop
+	bit<32> psa_ingress_output_metadata_egress_port
+	bit<32> local_metadata_depth1
+	bit<32> local_metadata_depth2
+	bit<32> local_metadata_depth3
+	bit<32> local_metadata_depth4
+	bit<32> Ingress_tmp
+	bit<32> Ingress_var
+}
+metadata instanceof user_meta_data_t
+
+action nonDefAct args none {
+	mov m.local_metadata_depth1 m.Ingress_var
+	xor m.local_metadata_depth1 0x2
```

Comment:
and missing for xor. Not required here?

Reply:
We are updating bit_size of field when it is not aligned. And i think, the overflow of arithmetic operation would be happening in case of Add, sub, shr and shl operation. Other binary operations may not result into overflow. So it may not be required here